web/pdf_viewer.js

```diff
@@ -798,13 +802,15 @@ var PDFViewer = (function pdfViewer() {
      * @param {PageViewport} viewport
      * @returns {TextLayerBuilder}
      */
-    createTextLayerBuilder: function (textLayerDiv, pageIndex, viewport) {
+    createTextLayerBuilder: function (textLayerDiv, pageIndex, viewport,
+                                      enhanceTextSelection) {
       return new TextLayerBuilder({
         textLayerDiv: textLayerDiv,
         eventBus: this.eventBus,
         pageIndex: pageIndex,
         viewport: viewport,
-        findController: this.isInPresentationMode ? null : this.findController
+        findController: this.isInPresentationMode ? null : this.findController,
+        enhanceTextSelection: enhanceTextSelection
```

Comment:
Nit: Since text selection isn't possible in Presentation Mode, it might be a bit more efficient to instead change this line to:
```avascript
enhanceTextSelection: this.isInPresentationMode ? false : enhanceTextSelection,
```